Coming up with some exercises for Appendix 1

Xintong-Li opened the pull request:
This PR is about some exercises for Appendix 1 as suggested by.

**Question 1**
![image](https://user-images.githubusercontent.com/35551659/87867973-de7c8500-c95f-11ea-90d4-321c0d0de34e.png)
This exercise is about recognizing linear and nonlinear equations. Note that the appearence of `\mathrm{e}` (as coefficient) will not change the linearity of the equation. In addition, for those linear equations, I ask students to write it in standard form (`a_1x_1+...+a_nx_n=b`). 

**Question 2**
![image](https://user-images.githubusercontent.com/35551659/87868000-1e436c80-c960-11ea-9b24-f0fdf02090bb.png)
This question is about solving system of linear equations and identify if a system is consistent or not. In particular, the solution to 2a is unique; 2b is inconsistant; 2c is consistant and has infinitely many solutions.

**Question 3**
![image](https://user-images.githubusercontent.com/35551659/87868080-b3466580-c960-11ea-9916-adbe9a79df0e.png)
![image](https://user-images.githubusercontent.com/35551659/87868084-c0635480-c960-11ea-9bf7-38e2986aa129.png)
This question is about coefficient/augmented matrix. Part 1 and 2 are misunderstanding of some students (in particular, me when I was in first year). Part 3 and 4 leads to a proof of the fact that a system of equations A is consistent iff Coe(A) and Aug(A) have the same rank. However, since this question uses the term rank, the proper place of it might be end-of-section exercises of Appendix 2 or Module 11.

**Question 4**
![image](https://user-images.githubusercontent.com/35551659/87868140-2e0f8080-c961-11ea-8ec7-06e18d2a35e4.png)
This question is about a classical ancient Chinese problem. The scenario is simple, and I think students can easily solve it using materials in Appendix 1 (the system representing this problem is consistent, and has a unique solution). However, then we introduced three variations of this problem so that students can analyze this real world problems using their linear equation knowledge:
The first variation makes the system inconsistent, so the variation has no answer.
The second variation makes the system consistent, but now it has infinitely many solutions. Thus, students need to learn that "solutions to real world problems are not always unique".
The third variations keeps the consistency of system and uniqueness of solution. but this time the solution is not integers. Thus, it does not satisfy the scenario of thie problem. Students need to recognize this, and explain why this solution is not valid.

JackC92 commented:
I think in Appendix 2, a footnote says augmented columns usually are not referred to as pivot columns. Perhaps Question 3.b needs some reconsideration?

siefkenj commented on book/modules/appendix1-exercises.tex:
```diff
@@ -1,7 +1,57 @@
 
 	\begin{problist}
-		\prob XXX Finish
+		\prob For each equation given below, determine if it is linear. If so, write it in its standard form. If not, explain what makes it nonlinear.
```
Run this file through https://siefkenj.github.io/latex-parser/ so that the lines are wrapped.

siefkenj replied:
linear -> linear equation.

Don't mention `standard form`. It's not important.

`\sec` and `\csc` and `\cot` should already be operators.

siefkenj commented on book/modules/appendix1-exercises.tex:
```diff
@@ -1,7 +1,57 @@
+      \item $5x+2y+8z=\cos(y)$
+    \end{enumerate}
+    
+    \prob For each system of linear equations given below, (a) write down its augmented matrix; (b) use row reduction algorithm to determine if it is consistent or not; (c) for each consistent system, give the complete solution set.
```
`complete solution`, or `solution set`, but not both. Change this to a `: (a) ...., (b) ...`. That is, a colon then a list separated by commas.

siefkenj commented on book/modules/appendix1-exercises.tex:
```diff
@@ -1,7 +1,57 @@
+      \item \systeme{x_1-x_2+5x_3+x_4=1, x_1+x_2-2x_3+3x_4=3, 3x_1-x_2+8x_3+x_4=5, x_1+3x_2-9x_3+7x_4=5}
+    \end{enumerate}
+    
+    \prob Let $\mathcal{A}$ be a system of linear equations with $n$ unknowns and $m$ equations. Kokoro is studying the condition for $\mathcal{A}$ to be consistent. Below, let $\operatorname{Coe}(\mathcal{A})$ be the coefficient matrix of \(\mathcal{A}\), and let $\operatorname{Aug}(\mathcal{A})$ be the augmented matrix of \(\mathcal{A}\).
```
This setup is too much and introducing new notation is confusing. I also worry that it's too abstract. Can you craft something involving specific 2x2 and 2x3 matrices (you could but the occasional unknown in a coefficient matrix)?

siefkenj commented on book/modules/appendix1-exercises.tex:
```diff
@@ -1,7 +1,57 @@
+    \end{enumerate}
+    
+    
+    \prob Here's a classical Chinese problem:
```
Cool :-). Is this written in any historical text? Can we get a citation?

siefkenj commented on book/modules/appendix1-exercises.tex:
```diff
@@ -1,7 +1,57 @@
+    
+    \prob Here's a classical Chinese problem:
+    
+    \emph{In the cage were chickens and rabbits. We had 35 in total, and together they have 94 legs. How many chickens and how many rabbits were in the cage?}
```
Let's make this a little more of a story. `On Tammy's farm, there is a cage with 35 animals. Some chickens and some rabbits. Tammy counted the total number of legs in the cage and found that there were 94 legs in all (notably, each chicken has exactly two legs and each rabbit has four legs). Tammy then decided to use this information to figure out how many chickens and how many rabbits there were.`

Xintong-Li commented:
Comments:
1. Now Problem 3 has been completely revised.

2. About the ancient Chinese problem:
The _Chicken and Rabbit in the Same Cage_ problem is first introduced in the ancient Chinese book _Mathematical classic of Master Sun_ (or _Sunzi Suanjing_, [here's the Wikipedia page](https://en.wikipedia.org/wiki/Sunzi_Suanjing)), where the configuration is the same as the one I gave here (total-35, leg-94, chicken-23, rabbit-12). The original occurrence of this problem is accompanied by a solution, which did not use any tool from linear algebra.

luema commented:
> **Question 2**
> ![image](https://user-images.githubusercontent.com/35551659/87868000-1e436c80-c960-11ea-9b24-f0fdf02090bb.png)

Instead of `(a) write down its augmented matrix, (b) use row reduction ...` I think it might be better to say `(i) write down its augmented matrix, (ii) use row reduction ...` since the systems are already denoted `(a)`, `(b)`, etc.

luema commented on book/modules/appendix1-exercises.tex:
```diff
@@ -1,7 +1,85 @@
+		\begin{enumerate}
+			\item Set up a system of linear equations representing this scenario.
+
+			\item Is the system consistent? If so, give the complete solution.
```
I think "`Is the system consistent? If so, give the complete solution.`" could be moved to part (a) above. 

So far we still haven't asked the reader to find exactly how many chickens and rabbits are in the cage, so maybe part (b) could ask that?

Xintong-Li replied:
I was considering "give the complete solution" as "solving the problem". But, yes, it's better if we add a part to ask them to solve the problem in terms of # of chickens and rabbits. (I think it can be another subpart of part(b) though.)

luema replied:
Yeah, I think subparts should work. There are a lot of interesting questions that could be asked under part (b) like `Besides the solution you found, are there any other combinations of chickens and rabbits that solves the system?` or `How do you know there can only be one possible combination of chickens and rabbits?` or `If the system were inconsistent, what would that mean about the information (total number of animals and total number of legs) Kokoro shared?`, but I'm not sure if those would be too difficult/too vague.

JackC92 commented on book/modules/appendix1-exercises.tex:
```diff
@@ -1,7 +1,85 @@
+
+			\item $\cos(4)x+\sin(4)y+\tan(4)z=\sec(4)x+\csc(4)y+\cot(4)z$
+
+			\item $5x+2y+8z=\cos(y)$
```
I think an equation like `e^x=2^{x+y}` could be a good question to ask here.

JackC92 commented on book/modules/appendix1-exercises.tex:
```diff
@@ -1,7 +1,85 @@
+
+		\prob Consider the matrix equation $M\vec x=\vec b$, where $M=\mat{2 & 8 \\ 4 & 16}$.
+		\begin{enumerate}
+			\item Show that with $\vec b=\mat{7 \\ 14}$, the system is
```
Maybe test the students' understanding of _consistent_ by asking a system with _one_ solution and another with _infinite_ solution. This can also test for quantifiers.

JackC92 commented on book/modules/appendix1-exercises.tex:
```diff
@@ -1,7 +1,85 @@
+			\item Come up with another $\vec b$ to make the system consistent.
+				Justify your answer.
+
+			\item Show that with $\vec b=\mat{5 \\ 12}$， the system is
```
The comma in this line is a Chinese character, which does not print when compiling.

Xintong-Li commented:
Thanks for everyone's comments. I have updated the exercise based on reviews. 
For your convinence, here I will outline my change toevery exercise.

**Exercise 1**
I included the interesting x/y=1.

**Exercise 2, Exercise 3**
Those two problems are newly added, discussing conversion between vector equations and linear systems.

**Exercise 4**
Part (c), (d), and (g) are newly added to address, respectively, the case of one single equation, one single variable, and the trivial equation (0x+0y+0z=0).

**Exercised 5**
I removed Part (e) which made use of the concept rank. I also changed the setting from matrix-vector multiplication to vector equation. 

**Exercise 6**
For Part (b), I included a subexercise to ask students to interpret their answer as the number of chickens and rabbits respectively (Thanks Samuel for his suggestion). Other than that, this problem is basically unchanged.

**Exercise 7**
This is a newly added T/F question. Besides the question about y=mx+c suggested by Shukui, I also added parts concerning consistency of linear equations.
![image](https://user-images.githubusercontent.com/35551659/88470353-3e77ac00-cec9-11ea-9567-8e32a1eb0b2b.png)
![image](https://user-images.githubusercontent.com/35551659/88470357-446d8d00-cec9-11ea-8c32-48970861ba9d.png)
